### ISSUE TITLE
Lacking arguments in array.sort()'s compareFn

### ISSUE THREAD

**Issue by Serifold** [javascript-tutorial/en.javascript.info#3562]
Hello, I found this weird usage in test [Shuffle an array](https://javascript.info/array-methods#shuffle-an-array)'s "simple solution":

![image](https://github.com/javascript-tutorial/en.javascript.info/assets/80401151/678c9a62-033f-40db-b9aa-2d7b57c62e3b)

There is just no arguments in the `array.sort()`'s `compareFn`
```js
array.sort(() => Math.random() - 0.5);
```

But [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#parameters) says the `compareFn` if we use `compareFn`, the arguments of `compareFn` "Will never be undefined.":

![image](https://github.com/javascript-tutorial/en.javascript.info/assets/80401151/27735ec7-2e97-4fb4-8aee-f9b2fa2d7d76)

According to MDN, the `compareFn` must be used with arguments, i.e.
```js
array.sort((a, b) => Math.random() - 0.5);
```

If it is a typo indeed, please fix it at your convenience, thank you.

**Comment by inishantjain** [comment]
In my opinion, the shuffle function has nothing to do with the elements of the array since it isn't actually sorting the array. Therefore, it does not need to be passed an argument.

**Comment by serifold** [comment]
> In my opinion, the shuffle function has nothing to do with the elements of the array since it isn't actually sorting the array. Therefore, it does not need to be passed an argument.

I don't think so. The shuffles function does sort the array actually, although not in a ascending or descending order which are the mostly used orders, but in a *random* order.
